app/lib/features/search/widgets/quick_jump.dart

```diff
@@ -56,12 +55,27 @@ class QuickJump extends ConsumerWidget {
                     onPressed: () {
                       navigateTo(route: Routes.pins);
                     },
-                    icon: const Icon(Atlas.pin_thin, size: 32),
+                    icon: const Icon(Atlas.pin_thin),
```

Comment:
```suggestion
                    icon: const Icon(Atlas.pin_thin, size: 32),
```

Pretty sure the size was set on purpose because otherwise this icon is crammed into the circle, appearing to pierce it:

![grafik](https://github.com/acterglobal/a3/assets/40496/8188bc9c-67c9-457e-ad17-cce4c7b8b876)